test/e2e/manifests/pkg/activation/automatic/configuration-1.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: e2e-activation
+spec:
+  # NOTE: This package is manually built and pushed to the registry.
+  package: xpkg.upbound.io/awg-test/e2e-activation:v0.1.0
```

Comment:
If someone with write access can please copy both `v0.1.0` and `v0.2.0` of this configuration to `xkpg.crossplane.io/crossplane/e2e-activation` I'll update the manifests :pray: